server/db/schema/students.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE Student (
+    id INTEGER NOT NULL PRIMARY KEY,
+    level ENUM('beginner', 'intermediate', 'advanced') NOT NULL,
```

Comment:
Can you roll this back to when you had LEVEL as a type instead of an inline enum?